src/richText.js

```diff
@@ -394,7 +394,7 @@ define([
 
         // only support absolute path right now
         if (!form.getMugByPath(xpath) && !/instance\('casedb'\)/.test(xpath)) {
-            return value.replace('<', '&lt;').replace('>', '&gt;');
+            return $('<span>').text(value);
```

Comment:
Is this supposed to create a bubble? (I think it does not from local testing) If not, should it say `$('<span>').text(value).html()`? If yes, should it have a `label-datanode` class so it gets handled by the bubble widget?

If I'm not mistaken, this creates markup in ckeditor along the lines of:
```
label text <span>&lt;output value="unknown(/data/text)"&gt;&lt;/output&gt;</span> more text
```
which appears in the label ckeditor text area like this:
```
label text <output value="unknown(/data/text)" /> more text
```
What happens if the label is edited and the first 19 characters are deleted so it looks like
```
value="unknown(/data/text)" /> more text
```
Is the span element still in there? Does it get handled properly on getting the value out of the editor? Looks like it does in my local tests, but not sure if it will in all cases.